### PR TITLE
Add dto for instrument

### PR DESCRIPTION
**Pull request by sven1103**

### CODE REVIEW

File: src/main/groovy/life/qbic/datamodel/dtos/imaging/Instrument.groovy [hunk 1]
```diff
@@ -1,12 +1,72 @@
...
- * @since <versiontag>
+ * @since 1.10.0
  */
 class Instrument {
+
+  /**
+   * The manufacturer of the instrument.
+   * Examples are 'Leica', 'Zeiss' and many more.
+   */
+  private final String manufacturer
```
Comment (jenniferboedker) [inline src/main/groovy/life/qbic/datamodel/dtos/imaging/Instrument.groovy:19]: do we want to have a controlled vocabulary here?

File: src/main/groovy/life/qbic/datamodel/dtos/imaging/Instrument.groovy [hunk 2]
```diff
@@ -1,12 +1,72 @@
...
+
+  /**
+   * The model type of the instrument.
+   */
+  private final String model
```
Comment (jenniferboedker) [inline src/main/groovy/life/qbic/datamodel/dtos/imaging/Instrument.groovy:24]: Also here a controlled vocabulary would make sense?

File: src/main/groovy/life/qbic/datamodel/dtos/imaging/Instrument.groovy [hunk 3]
```diff
@@ -1,12 +1,72 @@
...
+   *
+   * @see: #Location
+   */
+  private final Location location
+
+  /**
+   * The type of the instrument.
+   * Example: 'Confocal Microscope'.
+   */
+  private final String type
```
Comment (jenniferboedker) [inline src/main/groovy/life/qbic/datamodel/dtos/imaging/Instrument.groovy:42]: controlled vocabulary?

File: src/main/groovy/life/qbic/datamodel/dtos/imaging/Instrument.groovy (hunk 1 above)
Comment (sven1103) [reply src/main/groovy/life/qbic/datamodel/dtos/imaging/Instrument.groovy:19]: The enumeration is unknown to me atm, so I used String

File: src/main/groovy/life/qbic/datamodel/dtos/imaging/Instrument.groovy (hunk 2 above)
Comment (sven1103) [reply src/main/groovy/life/qbic/datamodel/dtos/imaging/Instrument.groovy:24]: The enumeration is unknown to me atm, so I used String

File: src/main/groovy/life/qbic/datamodel/dtos/imaging/Instrument.groovy (hunk 3 above)
Comment (sven1103) [reply src/main/groovy/life/qbic/datamodel/dtos/imaging/Instrument.groovy:42]: The enumeration is unknown to me atm, so I used String

**Pull request merged**